app/src/lib/design-system-v2/tokens/colors.ts

```diff
@@ -48,6 +48,17 @@ const DEFAULT_MAP_TOKENS: MapTokens = {
   "primary-800": "#001F88",
   "primary-900": "#001670",
   "primary-1000": "#111A2C",
+  "secondary-0": "#EEDCFA",
+  "secondary-100": "#E2COFA",
+  "secondary-200": "#D5A3FB",
+  "secondary-300": "#C686FA",
+  "secondary-400": "#B568F9",
+  "secondary-500": "#A447F8",
+  "secondary-600": "#750FCD",
+  "secondary-700": "#680CB7",
+  "secondary-800": "#2B084B",
+  "secondary-900": "#420975",
+  "secondary-1000": "#240342",
   "neutral-0": "#E9E9E9",
   "neutral-100": "#E9E9E9",
   "neutral-200": "#D1D1D1",
```

Comment:
@wrongsahil could you please review the PR for this here - https://github.com/requestly/requestly/pull/1942

_couldn't reopen this PR since this PR was squash merged_